### PR TITLE
Update ironic-inspector-images for 4.8

### PR DESCRIPTION
**Pull request by elfosardo**
Also update 4.7 and 4.6 prevalidation jobs.

### CODE REVIEW

**Comment by elfosardo** [comment]
@smarterclayton @petr-muller can you please have a look at this? It will help us a lot moving forward with the work on 4.8 and backports to 4.7 and 4.6

**Review by petr-muller — APPROVED** [review]
Approved and held - please remove the `ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml` change and feel free to `/lgtm` to merge.

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml [hunk 1]
```diff
@@ -161,7 +161,7 @@ periodics:
     job-impl.setup-pod: launch-osd-create-create
     job-impl.workload-container: test
     job-impl.workload-pod: launch-wait
-  cluster: build01
+  cluster: build02
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:164]: /hold

Please remove this change, it's a flake (being fixed already).

**Review by petr-muller — APPROVED** [review]

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml (hunk 1 above)
Comment (elfosardo) [reply ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:164]: will do, thanks!

**Comment by elfosardo** [comment]
ci/rehearse/openshift/ironic-inspector-image/release-4.8/e2e-metal-ipi is failing because of Packet issues, and it's the same test as ci/rehearse/openshift/ironic-inspector-image/master/e2e-metal-ipi , which is green, so I strongly believe this can be merged

**Comment by derekhiggins** [comment]
> ci/rehearse/openshift/ironic-inspector-image/release-4.8/e2e-metal-ipi is failing because of Packet issues, and it's the same test as ci/rehearse/openshift/ironic-inspector-image/master/e2e-metal-ipi , which is green, so I strongly believe this can be merged

Should we remove the test if its redundant?

**Comment by elfosardo** [comment]
> > ci/rehearse/openshift/ironic-inspector-image/release-4.8/e2e-metal-ipi is failing because of Packet issues, and it's the same test as ci/rehearse/openshift/ironic-inspector-image/master/e2e-metal-ipi , which is green, so I strongly believe this can be merged
> 
> Should we remove the test if its redundant?

no, not really, it needs to be there for the fast forward merges anyway and to be ready for the 4.8 release
all the ipi metal jobs are not even required here, they're here for us to double-check

**Review by dtantsur — APPROVED** [review]
/lgtm

**Pull request merged**